### PR TITLE
chore: updating windows VHD to include 3B patches + march k8s packages

### PR DESCRIPTION
**Pull request by marosset**
**Reason for Change**:

**Issue Fixed**:

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: vhd/packer/configure-windows-vhd.ps1 [hunk 1]
```diff
@@ -77,11 +77,16 @@ function Get-FilesToCacheOnVHD
...
             "https://kubernetesartifacts.azureedge.net/kubernetes/v1.15.9/windowszip/v1.15.9-1int.zip",
             "https://kubernetesartifacts.azureedge.net/kubernetes/v1.15.10/windowszip/v1.15.10-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.15.11/windowszip/v1.15.11-1int.zip",
             "https://kubernetesartifacts.azureedge.net/kubernetes/v1.16.6/windowszip/v1.16.6-1int.zip",
             "https://kubernetesartifacts.azureedge.net/kubernetes/v1.16.7/windowszip/v1.16.7-1int.zip",
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.16.8/windowszip/v1.16.8-1int.zip",
             "https://kubernetesartifacts.azureedge.net/kubernetes/v1.17.2/windowszip/v1.17.2-1int.zip",
             "https://kubernetesartifacts.azureedge.net/kubernetes/v1.17.3/windowszip/v1.17.3-1int.zip",
-            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.18.0-beta.1/windowszip/v1.18.0-beta.1-1int.zip"
+            "https://kubernetesartifacts.azureedge.net/kubernetes/v1.17.4/windowszip/v1.17.4-1int.zip",
```
Comment (jackfrancis) [inline vhd/packer/configure-windows-vhd.ps1]: Are we intentionally leaving 3 versions in here?

Comment (jackfrancis) [reply vhd/packer/configure-windows-vhd.ps1]: 3 patch releases per minor version, I should say

Comment (marosset) [reply vhd/packer/configure-windows-vhd.ps1]: I think not keeping older versions is negatively impacting AKS cluster creation times.

Comment (marosset) [reply vhd/packer/configure-windows-vhd.ps1]: I don't have any data to back that up tho.

Comment (jackfrancis) [reply vhd/packer/configure-windows-vhd.ps1]: Being conservative is fine w/ me

**Pull request merged**